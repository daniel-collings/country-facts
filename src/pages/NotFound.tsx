import { NoSymbolIcon } from '@heroicons/react/24/solid'

export default function NotFound() {
  return (
    <div className="alert alert-info">
      <NoSymbolIcon className="h-8 w-8" />
      <span>This is not the page you're looking for.</span>
    </div>
  )
}
