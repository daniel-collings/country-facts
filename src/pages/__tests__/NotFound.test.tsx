import { describe, expect, test } from 'vitest'
import { render } from '@/utils/testUtils.tsx'
import NotFound from '@/pages/NotFound.tsx'

describe('NotFound Page', () => {
  test('should render correctly', () => {
    const { asFragment } = render(<NotFound />)
    expect(asFragment()).toMatchSnapshot()
  })
})
