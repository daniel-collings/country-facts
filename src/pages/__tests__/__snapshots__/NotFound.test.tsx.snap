// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound Component > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="h-full min-h-screen bg-base-100 p-4 sm:p-8"
    data-theme="light"
  >
    <main
      class="max-w-7xl mx-auto px-0 sm:px-6 lg:px-8"
    >
      <div
        class="alert alert-info"
      >
        <svg
          aria-hidden="true"
          class="h-8 w-8"
          data-slot="icon"
          fill="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="m6.72 5.66 11.62 11.62A8.25 8.25 0 0 0 6.72 5.66Zm10.56 12.68L5.66 6.72a8.25 8.25 0 0 0 11.62 11.62ZM5.105 5.106c3.807-3.808 9.98-3.808 13.788 0 3.808 3.807 3.808 9.98 0 13.788-3.807 3.808-9.98 3.808-13.788 0-3.808-3.807-3.808-9.98 0-13.788Z"
            fill-rule="evenodd"
          />
        </svg>
        <span>
          This is not the page you're looking for.
        </span>
      </div>
    </main>
  </div>
</DocumentFragment>
`;

exports[`NotFound Page > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="h-full min-h-screen bg-base-100 p-4 sm:p-8"
    data-theme="light"
  >
    <main
      class="max-w-7xl mx-auto px-0 sm:px-6 lg:px-8"
    >
      <div
        class="alert alert-info"
      >
        <svg
          aria-hidden="true"
          class="h-8 w-8"
          data-slot="icon"
          fill="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="m6.72 5.66 11.62 11.62A8.25 8.25 0 0 0 6.72 5.66Zm10.56 12.68L5.66 6.72a8.25 8.25 0 0 0 11.62 11.62ZM5.105 5.106c3.807-3.808 9.98-3.808 13.788 0 3.808 3.807 3.808 9.98 0 13.788-3.807 3.808-9.98 3.808-13.788 0-3.808-3.807-3.808-9.98 0-13.788Z"
            fill-rule="evenodd"
          />
        </svg>
        <span>
          This is not the page you're looking for.
        </span>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
